admin_guide/backup_restore.adoc

```diff
@@ -200,6 +200,77 @@ that {product-title} was previously installed.
 $ oc create -f mycluster.template.yaml
 ----
 
+[[backup-restore-recovering-etcd-hosts]]
+== Recovering etcd Hosts
+
+In cases where etcd hosts have failed, but you have at least one host still running, you can use that one host to recover etcd hosts without downtime.
+
+[NOTE]
+====
+The following presumes you have backups of the */etc/etcd* configuration for the etcd hosts.
+====
+
+. Add the desired number of hosts to the cluster.
+
+. Upgrade etcd in the surviving node with the provided version:
++
+----
+# rpm -Fvh etcd-2.3.7-3.el7.bz1350875.1.x86_64.rpm
+----
+
+. Create the new single node cluster using etcd’s --force-new-cluster option. You can do this with a long complex command using the values from /etc/etcd/etcd.conf, or you can temporarily modify the systemd unit file and start the service normally.
++
+To do so, edit the /usr/lib/systemd/system/etcd.service and add --force-new-cluster:
++
+----
+# sed -i '/ExecStart/s/"$/  --force-new-cluster"/' /usr/lib/systemd/system/etcd.service
+# cat /usr/lib/systemd/system/etcd.service  | grep ExecStart
+
+ExecStart=/bin/bash -c "GOMAXPROCS=$(nproc) /usr/bin/etcd --force-new-cluster"
+----
+
+. Remove `--force-new-cluster` from the systemd unit file.
+
+. Install the etcd rpm on the new hosts:
++
+----
+# rpm -ivh etcd-2.3.7-3.el7.bz1350875.1.x86_64.rpm
+----
+
+. Restore */etc/etcd* from the backup onto the new hosts.
```

Comment:
We may want to have docs / notes on ways to do this? Do we already have admin docs on how to backup or store data like this (such as store this on a ISCSI (RAID)?